--- ./MDB-0.9.8/MDB_pear_wrapper_test.php.org	Thu Jul  4 23:06:59 2002
+++ ./MDB-0.9.8/MDB_pear_wrapper_test.php	Thu Jul  4 23:20:11 2002
@@ -4,8 +4,8 @@
 // MDB test script for the PEAR DB Wrapper.
 //
 
-    require_once("pear_wrapper.php");
-    require_once("Var_Dump.php");
+    require_once("MDB/pear_wrapper.php");
+    require_once("MDB/MDB_Var_Dump.php");
 
     // just for kicks you can mess up this part to see some pear error handling
     $user = 'metapear';
@@ -23,7 +23,7 @@
     $db = DB::connect($dsn);
     // With DB::isError you can differentiate between an error or
     // a valid connection.
-    //echo Var_Dump::display($db)."<br>";
+    //echo MDB_Var_Dump::display($db)."<br>";
     if (DB::isError($db)) {
         die ($db->getMessage());
     }
@@ -36,22 +36,22 @@
     // run the query and get a result handler
     $result = $db->simpleQuery($query);
     echo "<br>tableInfo() ";
-    Var_Dump::display($db->tableInfo($result));
+    MDB_Var_Dump::display($db->tableInfo($result));
 
     $result = $db->query($query);
     echo "<br>numCols() ";
-    Var_Dump::display($result->numCols());
+    MDB_Var_Dump::display($result->numCols());
     $result->fetchInto(&$arr);
     echo "<br>fetchInto() ";
-    Var_Dump::display($arr);
+    MDB_Var_Dump::display($arr);
     echo "<br>free() ";
-    Var_Dump::display($result->free());
+    MDB_Var_Dump::display($result->free());
 
     $result = $db->query($query);
     echo "<br>numRows() ";
-    Var_Dump::display($result->numRows());
+    MDB_Var_Dump::display($result->numRows());
     echo "<br>fetchRow() ";
-    Var_Dump::display($result->fetchRow());
+    MDB_Var_Dump::display($result->fetchRow());
 
     // lets create a sequence on demand
     echo "<br>get the next id using on demand:<br>";
@@ -91,16 +91,16 @@
     echo "<br>see getOne in action:<br>".$db->getOne("SELECT trans_en FROM numbers WHERE number = ?",$array)."<br>";
     // You can disconnect from the database with:
     echo "<br>see getRow in action:<br>";
-    echo Var_Dump::display($db->getRow("SELECT * FROM numbers WHERE number = ?",$array))."<br>";
+    echo MDB_Var_Dump::display($db->getRow("SELECT * FROM numbers WHERE number = ?",$array))."<br>";
     echo "<br>see getCol in action:<br>";
-    echo Var_Dump::display($db->getCol("SELECT * FROM numbers", 1))."<br>";
+    echo MDB_Var_Dump::display($db->getCol("SELECT * FROM numbers", 1))."<br>";
     echo "<br>see getAll in action:<br>";
-    echo Var_Dump::display($db->getAll("SELECT * FROM test"))."<br>";
+    echo MDB_Var_Dump::display($db->getAll("SELECT * FROM test"))."<br>";
     echo "<br>see getAssoc in action:<br>";
-    echo Var_Dump::display($db->getAssoc("SELECT * FROM test", FALSE, "", DB_FETCHMODE_ASSOC))."<br>";
+    echo MDB_Var_Dump::display($db->getAssoc("SELECT * FROM test", FALSE, "", DB_FETCHMODE_ASSOC))."<br>";
     echo "tableInfo on a string:<br>";
-    echo Var_Dump::display($db->tableInfo("numbers"))."<br>";
+    echo MDB_Var_Dump::display($db->tableInfo("numbers"))."<br>";
     echo "<br>just a simple delete query:<br>";
-    echo Var_Dump::display($db->query("DELETE FROM numbers"))."<br>";
+    echo MDB_Var_Dump::display($db->query("DELETE FROM numbers"))."<br>";
     $db->disconnect();
 ?>
--- ./MDB-0.9.8/MDB_test.php.org	Thu Jul  4 23:07:10 2002
+++ ./MDB-0.9.8/MDB_test.php	Thu Jul  4 23:36:41 2002
@@ -6,9 +6,9 @@
 
     // MDB.php doesnt have to be included since manager.php does that
     // manager.php is only necessary for handling xml schema files
-    require_once("manager.php");
+    require_once("MDB/manager.php");
     // only including this to output result data
-    require_once("Var_Dump.php");
+    require_once("MDB/Var_Dump.php");
 
     // just for kicks you can mess up this part to see some pear error handling
     $user = 'metapear';
@@ -69,41 +69,41 @@
     // lets just get row:0 and free the result
     $db->fetchRow($result, $array);
     echo "<br>row:<br>";
-    echo Var_Dump::display($array)."<br>";
+    echo MDB_Var_Dump::display($array)."<br>";
     // run the query and get a result handler
     $result = $db->query($query);
     // lets just get column:0 and free the result
     $db->fetchCol($result, $array);
     echo "<br>column:<br>";
-    echo Var_Dump::display($array)."<br>";
+    echo MDB_Var_Dump::display($array)."<br>";
     // run the query and get a result handler
     $result = $db->query($query);
     // lets just get column:0 and free the result
     $db->fetchCol($result, $array, DB_FETCHMODE_DEFAULT, 2);
     echo "<br>get column #2 (counting from 0):<br>";
-    echo Var_Dump::display($array)."<br>";
+    echo MDB_Var_Dump::display($array)."<br>";
     // run the query and get a result handler
     $result = $db->query($query);
     echo "tableInfo:<br>";
-    echo Var_Dump::display($db->tableInfo($result))."<br>";
+    echo MDB_Var_Dump::display($db->tableInfo($result))."<br>";
     // lets just get everything and free the result
     $result = $db->query($query);
     $types = array('integer', 'text', 'timestamp');
     $db->setResultTypes($result, $types);
     $db->fetchAll($result, &$array);
     echo "<br>all:<br>";
-    echo Var_Dump::display($array)."<br>";
+    echo MDB_Var_Dump::display($array)."<br>";
     // save some time with this function
     // lets just get all and free the result
     $db->queryAll($query, $array);
     echo "<br>all with just one call:<br>";
-    echo Var_Dump::display($array)."<br>";
+    echo MDB_Var_Dump::display($array)."<br>";
     // run the query with the offset 1 and count 1 and get a result handler
     $result = $db->limitQuery($query, 1, 1);
     // lets just get everything but with an associative array and free the result
     $db->fetchAll($result, $array, DB_FETCHMODE_ASSOC);
     echo "<br>associative array with offset 1 and count 1:<br>";
-    echo Var_Dump::display($array)."<br>";
+    echo MDB_Var_Dump::display($array)."<br>";
     // lets create a sequence
     echo "<br>create a new seq with start 3 name real_funky_id<br>";
     $err = $db->createSequence("real_funky_id",3);
@@ -134,35 +134,35 @@
     );
     $prepared_query = $db->prepareQuery("INSERT INTO numbers VALUES(?,?,?)");
     echo "running executeMultiple<br>";
-    echo Var_Dump::display($db->executeMultiple($prepared_query, NULL, $alldata))."<br>";
+    echo MDB_Var_Dump::display($db->executeMultiple($prepared_query, NULL, $alldata))."<br>";
     echo "<br>affected rows:<br>";
     echo $db->affectedRows()."<br>";
     $array = array(4);
     echo "<br>see getOne in action:<br>";
-    echo Var_Dump::display($db->getOne("SELECT trans_en FROM numbers WHERE number = ?",'text',$array))."<br>";
+    echo MDB_Var_Dump::display($db->getOne("SELECT trans_en FROM numbers WHERE number = ?",'text',$array))."<br>";
     // You can disconnect from the database with:
     echo "<br>see getRow in action:<br>";
-    echo Var_Dump::display($db->getRow("SELECT * FROM numbers WHERE number = ?",array('integer','text','text'),$array))."<br>";
+    echo MDB_Var_Dump::display($db->getRow("SELECT * FROM numbers WHERE number = ?",array('integer','text','text'),$array))."<br>";
     echo "<br>see getCol in action:<br>";
-    echo Var_Dump::display($db->getCol("SELECT * FROM numbers",'text', 1))."<br>";
+    echo MDB_Var_Dump::display($db->getCol("SELECT * FROM numbers",'text', 1))."<br>";
     echo "<br>see getAll in action:<br>";
-    echo Var_Dump::display($db->getAll("SELECT * FROM test",array('integer','text','text')))."<br>";
+    echo MDB_Var_Dump::display($db->getAll("SELECT * FROM test",array('integer','text','text')))."<br>";
     echo "<br>see getAssoc in action:<br>";
-    echo Var_Dump::display($db->getAssoc("SELECT * FROM test",array('integer','text','text'), NULL, NULL, DB_FETCHMODE_ASSOC))."<br>";
+    echo MDB_Var_Dump::display($db->getAssoc("SELECT * FROM test",array('integer','text','text'), NULL, NULL, DB_FETCHMODE_ASSOC))."<br>";
     echo "tableInfo on a string:<br>";
-    echo Var_Dump::display($db->tableInfo("numbers"))."<br>";
+    echo MDB_Var_Dump::display($db->tableInfo("numbers"))."<br>";
     echo "<br>just a simple delete query:<br>";
-    echo Var_Dump::display($db->query("DELETE FROM numbers"))."<br>";
+    echo MDB_Var_Dump::display($db->query("DELETE FROM numbers"))."<br>";
     // subselect test
     $sub_select = $db->subSelect("SELECT test_name from test WHERE test_name = ".$db->getTextFieldValue('gummihuhn'), TRUE);
-    echo Var_Dump::display($sub_select)."<br>";
+    echo MDB_Var_Dump::display($sub_select)."<br>";
     $query_with_subselect = "SELECT * FROM test WHERE test_name IN (".$sub_select.")";
     // run the query and get a result handler
     echo $query_with_subselect."<br>";
     $result = $db->query($query_with_subselect);
     $db->fetchAll($result, &$array);
     echo "<br>all with subselect:<br>";
-    echo Var_Dump::display($array)."<br>";
+    echo MDB_Var_Dump::display($array)."<br>";
   
     /*
     // ok now lets create a new xml schema file from the existing DB
@@ -171,7 +171,7 @@
     $manager->setupDatabase($dsn);
     $manager->getDefinitionFromDatabase();
     // this is the database definition as an array
-    echo Var_Dump::display($manager->database_definition)."<br>";
+    echo MDB_Var_Dump::display($manager->database_definition)."<br>";
 
     // new we will write this array as an xml schema file
     $manager->debug = "Output";
--- ./MDB-0.9.8/Var_Dump.php.org	Thu Jul  4 23:07:42 2002
+++ ./MDB-0.9.8/Var_Dump.php	Thu Jul  4 23:34:44 2002
@@ -73,7 +73,7 @@
 * - If given an object, informations about the object and the class
 * are printed.
 */
-class Var_Dump extends PEAR
+class MDB_Var_Dump extends PEAR
 {
 
 
--- ./MDB-0.9.8/driver_test.php.org	Thu Jul  4 23:07:55 2002
+++ ./MDB-0.9.8/driver_test.php	Thu Jul  4 23:11:51 2002
@@ -68,9 +68,9 @@
  */
 
 
-require("metabase_wrapper.php");
-require("manager.php");
-require("lob.php");
+require("MDB/metabase_wrapper.php");
+require("MDB/manager.php");
+require("MDB/lob.php");
 
 Function VerifyFetchedValues($database,$result,$row,&$data,&$value,&$field)
 {
--- ./MDB-0.9.8/manager.php.org	Thu Jul  4 23:08:47 2002
+++ ./MDB-0.9.8/manager.php	Thu Jul  4 23:13:01 2002
@@ -47,9 +47,9 @@
 // MDB XML Schema Manager
 //
 
-require_once(dirname(__FILE__)."/MDB.php");
-require_once(dirname(__FILE__)."/parser.php");
-require_once(dirname(__FILE__)."/xml_parser.php");
+require_once("MDB/MDB.php");
+require_once("MDB/parser.php");
+require_once("MDB/xml_parser.php");
 
 /**
 * The database manager is a class that provides a set of database 
--- ./MDB-0.9.8/metabase_wrapper.php.org	Thu Jul  4 23:08:59 2002
+++ ./MDB-0.9.8/metabase_wrapper.php	Thu Jul  4 23:21:52 2002
@@ -47,7 +47,7 @@
 // Metabase Wrapper for MDB.
 //
 
-require_once("MDB.php");
+require_once("MDB/MDB.php");
 
 $metabase_databases = &$databases;
 $metabases_lobs = &$lobs;
@@ -1274,7 +1274,7 @@
     }
 }
 
-class Metabase_manager_class
+class MDB_Metabase_manager_class
 {
     var $MDB_manager_object;
     
--- ./MDB-0.9.8/mysql.php.org	Thu Jul  4 23:33:38 2002
+++ ./MDB-0.9.8/mysql.php	Thu Jul  4 23:14:13 2002
@@ -50,7 +50,7 @@
  if (!defined("MDB_MYSQL_INCLUDED")) {
     define("MDB_MYSQL_INCLUDED", 1);
 
-require_once "common.php";
+require_once "MDB/common.php";
  
 class MDB_mysql extends MDB_common
 {
--- ./MDB-0.9.8/pear_wrapper.php.org	Thu Jul  4 23:09:13 2002
+++ ./MDB-0.9.8/pear_wrapper.php	Thu Jul  4 23:28:49 2002
@@ -47,7 +47,7 @@
 // PEAR DB Wrapper for MDB.
 //
  
-require_once "MDB.php";
+require_once "MDB/MDB.php";
 
 class DB
 {
@@ -106,7 +106,7 @@
     }
 }
 
-class DB_Error extends PEAR_Error
+class MDB_DB_Error extends PEAR_Error
 {
     function DB_Error($code = DB_ERROR, $mode = PEAR_ERROR_RETURN,
               $level = E_USER_NOTICE, $debuginfo = NULL)
@@ -119,7 +119,7 @@
     }
 }
 
-class DB_Warning extends PEAR_Error
+class MDB_DB_Warning extends PEAR_Error
 {
     function DB_Warning($code = DB_WARNING, $mode = PEAR_ERROR_RETURN,
             $level = E_USER_NOTICE, $debuginfo = NULL)
@@ -132,7 +132,7 @@
     }
 }
 
-class DB_result
+class MDB_DB_result
 {
     var $dbh;
     var $result;
@@ -200,7 +200,7 @@
     }
 }
 
-class DB_row
+class MDB_DB_row
 {
     function DB_row(&$arr)
     {
--- ./MDB-0.9.8/pgsql.php.org	Thu Jul  4 23:09:21 2002
+++ ./MDB-0.9.8/pgsql.php	Thu Jul  4 23:15:43 2002
@@ -51,7 +51,7 @@
     define("MDB_PGSQL_INCLUDED", 1);
 
 
-require_once "common.php";
+require_once "MDB/common.php";
 
 class MDB_pgsql extends MDB_common
 {
--- ./MDB-0.9.8/setup_test.php.org	Thu Jul  4 23:09:36 2002
+++ ./MDB-0.9.8/setup_test.php	Thu Jul  4 23:16:09 2002
@@ -47,8 +47,8 @@
 // Metabase test suite.
 //
 
-    require("manager.php");
-    require("metabase_wrapper.php");
+    require("MDB/manager.php");
+    require("MDB/metabase_wrapper.php");
 
 Function Output($message)
 {
--- ./MDB-0.9.8/xml_parser.php.org	Thu Jul  4 23:39:33 2002
+++ ./MDB-0.9.8/xml_parser.php	Thu Jul  4 23:31:47 2002
@@ -79,7 +79,7 @@
         $xml_parser_handlers[$parser]->CharacterData($xml_parser_handlers[$parser],$data);
 }
 
-class xml_parser_handler_class
+class MDB_xml_parser_handler_class
 {
     var $xml_parser;
     var $error_number=0;
@@ -166,7 +166,7 @@
     }
 };
 
-class xml_parser_class
+class MDB_xml_parser_class
 {
     var $xml_parser=0;
     var $parser_handler;
@@ -437,7 +437,7 @@
                 return($error);
         }
     }
-    $parser=new xml_parser_class;
+    $parser=new MDB_xml_parser_class;
     $parser->store_positions=$store_positions;
     $parser->case_folding=$case_folding;
     $parser->target_encoding=$target_encoding;
