--- ./driver_test.php.org	Wed Jul 17 17:34:08 2002
+++ ./driver_test.php	Wed Jul 17 17:39:51 2002
@@ -148,7 +148,7 @@
 
     set_time_limit(0);
     $input_file="driver_test.schema";
-    $manager=new metabase_manager_class;
+    $manager=new Metabase_manager_class;
     $success=$manager->UpdateDatabase($input_file,$input_file.".before",$driver_arguments,$database_variables);
     $debug_output="";
     if(count($manager->warnings)>0)
--- ./setup_test.php.org        Wed Jul 10 13:04:11 2002
+++ ./setup_test.php    Wed Jul 17 17:44:44 2002
@@ -70,7 +70,7 @@
         "Password"=>"funky",
         "Debug"=>"Output"
     );
-    $manager=new metabase_manager_class;
+    $manager=new Metabase_manager_class;
     $manager->debug="Output";
     $success=$manager->UpdateDatabase($input_file,$input_file.".before",$arguments,$variables);
     if($success)

